### DISCUSSION TITLE
Ignore folder when generating Breadcrumbs?

### DISCUSSION THREAD

**Discussion by gudstrand** [SkepticMystic/breadcrumbs#358] [Q&A]
I have set breadcrumb info in my templates which works great, however I end up with the templates being listed as siblings via an implied relationship. Is there a way to exclude documents in designated folders to be ignored?

Thanks!

**Comment by dannyrappleyea** [comment]
+1 on ignoring the templates folder. I'm trying codeblocks like this to get the same effect, but a global option would be more convenient.

```
type: tree
dir: down
depth: -1
from: -"_templates"
```

**Comment by michaelpporter** [comment]
v4 ships this: **Settings → General → Excluded folders** (`exclude_folders`). Folders listed there are filtered out at the file-collection stage, so their notes are skipped as edge sources entirely. Closing as resolved — thanks!
